fe/fe-core/src/test/resources/sql/tpchcost/q20.sql

```diff
@@ -333,12 +333,12 @@ column statistics:
     "test.nation": {
       "nation": 25
     },
-    "test.lineitem": {
-      "lineitem": 600000000
-    },
     "test.partsupp": {
       "partsupp": 80000000
     },
+    "test.lineitem": {
```

Comment:
why these changed

Reply:
I guess because I modify the useage of DumpLog